djangocms_admin_style/sass/components/_modal.scss

```diff
@@ -57,7 +57,7 @@
                 // make sure that changelist form is floated left next to filter #238
                 #changelist-form {
                     float: left;
-                    width: calc(100% - 220px) !important;
+                    width: calc(100% - 248px) !important;
```

Comment:
oh, okay. i don't know how width of the sidebar is defined, but maybe we can base it off that?